index.html

```diff
@@ -49,7 +49,32 @@ <h2>Connect</h2>
        <li>https://github.com/curiousBellyButton</li>
      </ul>
   </section>
-  <footer></footer>
+  <section>
+    <h2>Leave a Message</h2>
+    <form id="leave_name" name="leave_message">
```

Comment:
```suggestion
    <form name="leave_message">
```

I don't think you reference this ID.

Reply:
at first I couldn't figure out how to reference "name" attr, so I added id, to easily use DOM Select, but then I found the way to do it, so I left it as is.